manifests/p/Pylogmon/pot/1.8.3/Pylogmon.pot.installer.yaml

```diff
@@ -0,0 +1,32 @@
+# Created with WinGet Releaser v2 using Komac v1.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Pylogmon.pot
+PackageVersion: 1.8.3
+InstallerLocale: zh-CN
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- silentWithProgress
+UpgradeBehavior: install
+Protocols:
+- '0'
+- '3'
+- GPL
+FileExtensions:
+- exe
+ReleaseDate: 2023-07-14
+AppsAndFeaturesEntries:
+- UpgradeCode: '{C4D134BA-2EFB-540F-A7FF-129FE538577E}'

```

Comment:
```suggestion
```
As far as I know, this value is for MSI installers and the EXE doesn't seem to be a bootstrap of MSI. I do wonder how you got this value 👀 Maybe it's a remnant of an older manifest that contained MSI installers?

Reply:
I think so. I switched the installation package from msi to exe and I didn't notice this option. Thank you for pointing it out.